src/gui/pages_menu/KM_GUIMenuMapEditor.pas

```diff
@@ -438,6 +438,7 @@ procedure TKMMenuMapEditor.ColumnClick(aValue: Integer);
 
 function TKMMenuMapEditor.GetMaps: TKMapsCollection;
 begin
+  Result := nil;
```

Comment:
I came to a better pattern:

case of
  1:
  2:
else
  raise Exception.Create();
end;

It keeps compiler happy, takes less LOC and catches errors well.